tests/interfaces/default_qubit_2_integration/test_autograd_qnode_default_qubit_2.py

```diff
@@ -50,6 +50,7 @@
     ["auto", DefaultQubit(), "adjoint", False, False],
     ["auto", DefaultQubit(), "spsa", False, False],
     ["auto", DefaultQubit(), "hadamard", False, False],
+    ["auto", qml.device("lightning.qubit", wires=5), "adjoint", False, True],
```

Comment:
fairly easy to add, nice!

Reply:
Might even be worth using this for `default.qubit.legacy` too.